src/main/java/core/basesyntax/service/impl/FruitTransactionParserServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitReport;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitTransactionParserService;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class FruitTransactionParserServiceImpl implements FruitTransactionParserService {
+    private final Map<String, Integer> storage = new HashMap<>();
```

Comment:
This map should be in a separate class `Storage`  in other package `core.basesyntax.db`, called fruits and it should be `public static final`